src/sentry/utils/dates.py

```diff
@@ -115,3 +116,19 @@ def parse_timestamp(value):
         except ValueError:
             rv = None
     return rv.replace(tzinfo=pytz.utc)
+
+def parse_stats_period(period):
```

Comment:
Pulled this out of @mattrobenolt PR (https://github.com/getsentry/sentry/pull/9622) into utils since we'll use this for Discover as well